app/src/main/java/org/mozilla/focus/notification/BrowsingNotificationService.java

```diff
@@ -97,6 +97,10 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                     activityIntent.setFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
                 }
 
+                // This doesn't seem to be needed on Android 7.1. It is needed on Android 6, or otherwise
+                // we crash with "AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag."
+                activityIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Technically the flag isn't needed as our activity is singleTask anyways. So maybe the check is now less strict on Android 7+.